### ISSUE TITLE
Calendar: Should be able to setup aria-label on input

### ISSUE THREAD

**Issue by Foxandxss** [primefaces/primeng#13319]
### Describe the bug

There are some componentes like `inputNumber` that will successfully redirect the `ariaLabel` `@Input()` to the component's input. In the case of the `Calendar` component, there is no way to set an `aria-label` in the input. You can add a `aria-labelledby` but that doesn't cover all cases. In fact, the docs on the component states that you can use `aria-label` and that is not currently true.

### Environment

Angular 16
Primeng 16.0.2

### Reproducer

Open any example of the calendar in stackblitz and check its accesibility

### Angular version

16.3

### PrimeNG version

16.0.2

### Build / Runtime

Angular CLI App

### Language

TypeScript

### Node version (for AoT issues node --version)

16.20.0

### Browser(s)

Chrome

### Expected behavior

ariaLabel should be a valid `@Input()` according to the docs and also to the other components.

**Comment by Foxandxss** [comment]
#6805 is an old issue.

**Comment by Foxandxss** [comment]
Fixed #13990
